Makefile

```diff
@@ -51,12 +51,12 @@ run: ## Run the web app
 	flask run -p 6011 --host=0.0.0.0
 
 .PHONY: run-celery
-run-celery: ## Run the celery workers
-	./scripts/run_celery.sh
+run-celery-local: ## Run the celery workers
+	./scripts/run_celery_local.sh
```

Comment:
We can keep the run-celery make target? we'd have to change/duplicate the `.PHONY` name too I think.

Reply:
These make targets were added in #1925 to make it clearer how to run locally, so I think we can just move the make targets to the new local scripts. Need tup update the `.PHONY`s though 👍